src/components/numeral-date/numeral-date.component.js

```diff
@@ -193,8 +199,18 @@ const NumeralDate = ({
           fieldHelp={fieldHelp}
           adaptiveLabelBreakpoint={adaptiveLabelBreakpoint}
           isRequired={required}
+          validationRedesignOptIn={validationRedesignOptIn}
           {...filterStyledSystemMarginProps(rest)}
         >
+          {validationRedesignOptIn && labelHelp && (
+            <StyledHintText>{labelHelp}</StyledHintText>
```

Comment:
`StyledHintText` and `ValidationMessage` components are meant to be placed in `FormField` in the future when we also redesign the other inputs?

Reply:
Yeah that would make sense to me 👍🏻